### DISCUSSION TITLE
Hallard LoRa Module

### DISCUSSION THREAD

**Discussion by timmbogner** [timmbogner/Farm-Data-Relay-System#152] [Q&A]
@aviateur17 
I'm helping a user that's using the Hallard module like you use, although it's fit onto a LOLIN32 Lite. They can run the old LoRa library okay but not RadioLib. I'm looking at the schematic for it and it is an unusual setup: the DIO pins are tied together, which shouldn't be an issue for now. But also, the SS and RST seem to be tied together? Was there anything special you needed to do to get that board going?

**Comment by aviateur17** [comment]
Yes, DIO0, DIO1, and DIO2 are all tied together to the same Pin via diodes.   I think in the program only DIO0 is being used so it's not an issue.   On the board there is a jumper solder pad called JP3 which defaults to SS being connected.  RST is not connected by default.   The RST pin is not required.

![image](https://user-images.githubusercontent.com/61305397/225503696-23bdb6f0-f27c-4229-8a85-e9e11137df47.png)

**Comment by aviateur17** [comment]
The only pins that are needed are:

LORA_SS
LORA_DIO
LORA_SPI_SCK
LORA_SPI_MISO
LORA_SPI_MOSI

**Reply by timmbogner** [reply]
Thanks! The jumper symbol threw me off.

**Reply by timmbogner** [reply]
Oh yeah, and also which RadioLib module do you use?

**Comment by aviateur17** [comment]
It's the HopeRFM95W which is basically the same as the SX1276 so I use the SX1276 config.
